src/components/screens/monitor/transactions/overview.js

```diff
@@ -0,0 +1,211 @@
+          className={`box-tabs ${styles.tabs}`}
+        />
+      </BoxHeader>
+      <div className={styles.container}>
+        <div className={`${styles.column} ${styles.pie}`}>
+          <h2 className={styles.title}>{t('Distribution of transaction types')}</h2>
+          <div className={styles.graph}>
+            <DoughnutChart
+              data={{
+                labels: transactionTypes
+                  .getListOf('title')
+                  .map(item => item
+                    .replace('Second passphrase registration', '2nd passphrase reg.')
+                    .replace('Multisignature creation', 'Multisig. creation')),
+                datasets: [
+                  {
+                    data: distributionByType,
+                  },
+                ],
+              }}
+            />
+          </div>
+        </div>
+        <div className={`${styles.column} ${styles.pie}`}>
+          <h2 className={styles.title}>{t('Amount per transaction (LSK)')}</h2>
+          <div className={styles.graph}>
+            <DoughnutChart
+              data={{
+                labels: Object.keys(distributionByAmount).map(range => formatNumberRange(range)),
+                datasets: [
+                  {
+                    data: Object.values(distributionByAmount),
+                  },
+                ],
+              }}
+            />
+          </div>
+        </div>
+        <div className={`${styles.column} ${styles.bar}`}>
+          <h2 className={styles.title}>{t('Transactions number / volume (LSK)')}</h2>
+          <div className={styles.graph}>
+            <BarChart
+              data={{
+                labels: txDateList.reverse(),
+                datasets: [
+                  {
+                    label: 'Number',
+                    yAxisID: 'Number',
+                    data: txCountList.reverse(),
+                    categoryPercentage: 0.5,
+                    barPercentage: 0.8,
+                  },
+                  {
+                    label: 'Volume',
+                    yAxisID: 'Volume',
+                    data: txVolumeList.reverse(),
+                    categoryPercentage: 0.5,
+                    barPercentage: 0.8,
+                  },
+                ],
+              }}
+              options={options}
+            />
+            <aside className={styles.legends}>
+              <h5 className={`${styles.legend} ${styles.number}`}>
+                <span>{t('Number')}</span>
+                <Tooltip className={`${styles.tooltip} showOnLeft`}>
+                  <p>{t('The number of transactions submitted on the given time period.')}</p>
+                </Tooltip>
+              </h5>
+              <h5 className={`${styles.legend} ${styles.volume}`}>
+                <span>{t('Volume')}</span>
+                <Tooltip className={`${styles.tooltip} showOnLeft`}>
+                  <p>{t('The aggregated of LSK volume transferred on the given time period.')}</p>
```

Comment:
I'm not sure where is this copy from but I like it more than the one in [design](https://projects.invisionapp.com/d/main#/console/18539694/390074223/preview). I just think that to be grammatically correct, it should say:

```suggestion
                  <p>{t('The aggregate of LSK volume transferred on the given time period.')}</p>
```